tools/scaling/sweep/node_to_node/unstructured_mesh/strong/generate_scaling_study.py

```diff
@@ -51,7 +51,7 @@ def generate_files_for_scaling_study(opensn_binary, gmsh_binary, geo_filename, n
 opensn_binary = "/path/to/opensn"
 
 # Path to the Gmsh binary
-gmsh_binary = "gmsh"
+gmsh_binary = "/Applications/Gmsh.app/Contents/MacOS/gmsh"
```

Comment:
Do you want to make this work only on macOS?

Reply:
Fixed.